doc/releases/changelog-0.11.0.md

```diff
@@ -102,7 +102,7 @@
   def subroutine():
       qml.Hadamard(wires=[0])
       qml.CNOT(wires=[0, 1])
-      qml.PauliX(wires=[1])
+      qml.X(1)
```

Comment:
We can keep all the historic changelog entries the same, except for the dev one.